contracts/BSCValidatorSet.sol

```diff
@@ -245,8 +260,12 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       if (validator.jailed) {
         emit deprecatedDeposit(valAddr,value);
       } else {
-        totalInComing = totalInComing.add(value);
-        validator.incoming = validator.incoming.add(value);
+        if (_isMigrated(index - 1)) {
+          IStakeHub(STAKE_HUB_ADDR).distributeReward{value: value}(valAddr);
```

Comment:
I think we still keep the reward within one day in this contract. If we `distributeReward` now, then, there is no way to perform similar felony and misdeamon to migrated validator.

Reply:
Done